deploy/helm/secret-operator/values.yaml

```diff
@@ -3,6 +3,14 @@
 image:
   repository: docker.stackable.tech/stackable/secret-operator
   pullPolicy: IfNotPresent
+csiProvisionerImage:
+  repository: docker.stackable.tech/sig-storage/csi-provisioner
+  tag: v3.1.0
+  pullPolicy: IfNotPresent
+csiNodeDriverRegistrarImage:
```

Comment:
Very minor nitpick: `I'd prefer to have these as csiProvisioner.image.repository` etc.